crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -59,13 +59,16 @@ Examples:
     #[arg(
         long = "rpc.cors-domains",
         long_help = r"Comma separated list of domains from which Cross-Origin requests will be accepted by the RPC server.
+
+Use '*' to indicate any domain and an empty list to disable CORS.
         
 Examples:
     single: http://one.io
     a list: http://first.com,http://second.com:1234
     any:    *",
```

Comment:
Ok, so this tiny example
```
Examples:
    single: http://one.io
    a list: http://first.com,http://second.com:1234
    any:    *" <-----------HERE
```
doesn't seem to do its work :shrug: / is not noticeable enough

Reply:
I did see it :) I just figured I should list all "exceptions" i.e. wildcard + empty list before examples already.